doc/mgr/dashboard.rst

```diff
@@ -1061,6 +1061,12 @@ location is::
 After running the above command, Ceph Dashboard is able to find the NFS-Ganesha
 configuration objects and we can start manage the exports through the Web UI.
 
+.. note::
+
+    Consider to use a separate pool for the NFS shares only. There is currently
+    a `know issue <https://tracker.ceph.com/issues/46176>`_ with listing the shares
+    if the NFS objects are stored together with a lot of other objects in one pool.
```

Comment:
Instead say:
```suggestion
    Separate pool for the NFS shares should be used. Otherwise it can cause this
    `known issue <https://tracker.ceph.com/issues/46176>`_ with listing of shares
    if the NFS objects are stored together with a lot of other objects in a single pool.
```